gems/pending/appliance_console/logfile_configuration.rb

```diff
@@ -0,0 +1,73 @@
+require 'active_record'
+require 'active_support/core_ext'
+require 'linux_admin'
+require 'pathname'
+require 'util/miq-password'
+require 'fileutils'
+require 'appliance_console/errors'
+require 'appliance_console/logging'
+require 'appliance_console/logical_disk_volume'
+require 'appliance_console/prompts'
+require 'appliance_console/utilities'
+
+RAILS_ROOT ||= Pathname.new(__dir__).join("../../../")
+
+module ApplianceConsole
+  class LogfileConfiguration
+    LOGFILE_DIRECTORY = Pathname.new("/var/www/miq/vmdb/log").freeze
+    LOGFILE_NAME = "miq_logs".freeze
+
+    attr_accessor :disk
+
+    include ApplianceConsole::Logging
+
+    def initialize
+      self.disk = nil
+    end
+
+    def activate
+      stop_evm
+      initialize_logfile_disk
+      restart_evm
+    end
+
+    def ask_questions
+      clear_screen
+      return false unless use_new_disk
+      choose_disk
+      true
+    end
+
+    private
+
+    def use_new_disk
+      agree("Configure a new logfile disk volume? (Y/N):")
+    end
+
+    def choose_disk
+      self.disk = ask_for_disk("logfile disk")
+    end
+
+    def initialize_logfile_disk
+      say 'Initializing logfile disk'
+      LogicalDiskVolume.new(:disk => disk, :mount_point => LOGFILE_DIRECTORY, :name => LOGFILE_NAME).setup
+      true
+    end
+
+    def restart_evm
```

Comment:
If this is just calling `.start` I would call it `start_evm` rather than `restart_evm`